src/GovUk.Education.ExploreEducationStatistics.Admin/Models/PreReleaseWindowStatus.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Models
+{
+    public class PreReleaseWindowStatus
+    {
+        [JsonConverter(typeof(StringEnumConverter))]
+        public PreReleaseAccess PreReleaseAccess { get; set; }
+
+        public DateTime PreReleaseWindowStartTime { get; set; }
```

Comment:
Not a big deal (and I can appreciate you didn't write it originally), but would be nicer if these properties weren't so verbose. We're in the context of a `PreReleaseWindowStatus` so prefixing everything with `PreRelease` is just words for the sake of it.

Would also require an update to the frontend code, but only in the `permissionService` as I've already put in some code to remove these prefixes.

Reply:
Thanks, I've shortened them now @ntsim.